extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -328,6 +350,11 @@ const OHIFCornerstoneViewport = React.memo(props => {
       Enums.Events.ELEMENT_ENABLED,
       elementEnabledHandler
     );
+    const elementToListen = elementRef.current;
```

Comment:
if this is for tests, I much rather have it inside cypress, since we have cornerstone on the window, you can grab it get the viewport, get the element, subscribe to it and just run a set interval until your closure state becomes true for the `dataRendered`

Reply:
Right now the primary use is for tests, but there are numerous other applications for which we need this sort of state:
* Clearing views when switching to a different image/view that is taking a while (to prevent hte user having the wrong image)